network/topics/scoring.go

```diff
@@ -33,16 +33,20 @@ func scoreInspector(logger *zap.Logger, scoreIdx peers.ScoreIndex) pubsub.Extend
 			filtered := make(map[string]*pubsub.TopicScoreSnapshot)
 			var totalInvalidMessages float64
 			var totalLowMeshDeliveries int
+			var totalMeshDeliveriesSquaredDeficitSum float64
+			meshDeliveryCounters := make([]float64, 0)
 			for topic, snapshot := range peerScores.Topics {
 				if snapshot.InvalidMessageDeliveries != 0 {
 					filtered[topic] = snapshot
 				}
 				if snapshot.InvalidMessageDeliveries > 0 {
 					totalInvalidMessages += math.Sqrt(snapshot.InvalidMessageDeliveries)
 				}
-				if snapshot.MeshMessageDeliveries < 107 {
+				if snapshot.MeshMessageDeliveries < 11.243655244810899 {
 					totalLowMeshDeliveries++
+					totalMeshDeliveriesSquaredDeficitSum += math.Pow(11.243655244810899-snapshot.MeshMessageDeliveries, 2)
```

Comment:
less my department but since you use this constant more than once better extact it as a `const`
I would even extract consts that I use only once

Reply:
This is just for testing. Will be dropped after everybody agrees with the logic.